module/VuFind/src/VuFind/Config/PathResolver.php

```diff
@@ -92,6 +97,67 @@ public function __construct(
         $this->localConfigDirStack = $localConfigDirStack;
     }
 
+    /**
+     * Get the config location based on the config name.
+     *
+     * @param string  $configName Config name
+     * @param ?string $path       path relative to VuFind base (optional; use null for
+     * default)
+     *
+     * @return ?ConfigLocationInterface
+     */
+    public function getConfigLocation(
+        string $configName,
+        ?string $path = null,
+    ): ?ConfigLocationInterface {
+        $localConfigLocation = $this->getLocalConfigLocation($configName, $path);
+        if ($localConfigLocation !== null) {
+            return $localConfigLocation;
+        }
+        return $this->getConfigLocationFromSpec($configName, $this->baseDirectorySpec, $path);
```

Comment:
Could be simplified with null coalescing:

```suggestion
        return $this->getLocalConfigLocation($configName, $path)
            ?? $this->getConfigLocationFromSpec($configName, $this->baseDirectorySpec, $path);
```